codespell_lib/data/dictionary.txt

```diff
@@ -18893,6 +18898,8 @@ itention->intention
 itentional->intentional
 itentionally->intentionally
 itentionaly->intentionally
+iteraction->interaction
+iteractions->interactions
 iteratable->iterable
```

Comment:
If you do not mind suggesting `iteration` as well?
```text
iteraction->interaction, iteration,
iteractions->interactions, iterations,
```

Reply:
[98365e9](https://github.com/codespell-project/codespell/pull/2423/commits/98365e91351c7cc1035d18c6eb7c849f2bf0b91b)